pkgs/yaml_edit/test/testdata/output/issue_55.golden

```diff
@@ -12,4 +12,4 @@ environment:
   sdk: ^3.0.0
 dependencies:
 dev_dependencies:
-  {}
+  {}
```

Comment:
Can we special case to not remove a final newline if there is one?

Reply:
If it's just for trailing empty block collections and trailing entries, sure. It's possible if that's an issue.

Reply:
Yeah, I think ideally we should preserve end-of-file newlines when possible.